package.json

```diff
@@ -1,12 +1,16 @@
 {
-  "name": "open-scd-core",
+  "name": "@openscd/open-scd-core",
   "version": "0.0.0",
+  "description": "The core editor component of open-scd, without any extensions pre-installed.",
+  "author": "Open-SCD",
   "license": "Apache-2.0",
   "packageManager": "npm@8.12.2",
   "type": "module",
+  "browser": "./foundation.js",
+  "main": "./foundation.js",
```

Comment:
it should be `main` _**or**_ `browser`, not both.

https://docs.npmjs.com/cli/v9/configuring-npm/package-json#browser

Reply:
removed browser